src/App.tsx

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
-import {Route,Routes} from "react-router-dom";
-import {LoginPage} from "./pages/LoginPage/LoginPage";
-
+import { Route, Routes } from 'react-router-dom';
+import { LoginPage } from './pages/LoginPage/LoginPage';
+import { Navigation } from './components/Navigation/Navigation';
+import { PortfolioPage } from './pages/PortfolioPage/PortfolioPage';
+import { AdminViewPage } from './pages/AdminViewPage/AdminViewPage';
 
 function App() {
-
   return (
-
-      <Routes>
-        <Route path={'/'} element={<LoginPage/>}/>
-      </Routes>
-
-  )
+    <Routes>
+      <Route path={'/'} element={<LoginPage />} />
+      <Route
+        path={'/portfolio'}
+        element={
+          <>
+            <Navigation />
+            <PortfolioPage />
+          </>
```

Comment:
Jestem zwolennikiem dawania pojedynczego elementu jako Route - wtedy ścieżki są bardziej klarowne, nie chciałbyś przenieść Navigation po prostu do PortfolioPage?

Reply:
to jest tylko tymczasowe rozwiązanie, docelowo navigation będzie wyświetlane dla wszystkich stron oprócz logowania także trzeba tu będzie dodać jakiś warunek przy jakich ścieżkach ma się nie wyświetlać.